test.js

```diff
@@ -46,6 +46,12 @@ test('unist-util-map', function (t) {
     'should work when passing an empty object'
   )
 
+  t.deepEqual(
+    map(u('root', [u('node', [u('leaf', '1')]), u('leaf', '2')]), asIs),
+    u('root', [u('node', [u('leaf', '1')]), u('leaf', '2')]),
+    'should keep node as-is'
+  )
+
```

Comment:
I'm not sure I follow how this change relates to adding the `Parent` or `MapFunction` back?

Reply:
I need to add a test case to ensure [`asIs`](https://github.com/syntax-tree/unist-util-map/pull/7/files#diff-13876b4beb64b9f156474dc78f9c923952a7ca210d4507b6b3135bbe244f8a60R79-R84) function is used so that eslint with not complain `no-unused-var`.